data/example/sample.ztxt

```diff
@@ -1,3 +1,3 @@
-{"key":{"entity_hash":3,"entity_id":"moe"} ,"value":{"default:injury": [ { "key":{"time": "2016-01-01", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "ear" , "severity": 1 } }]} , { "key":{"time": "2016-01-02", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "hair" , "severity": 2 } }]} ]}}
+{"key":{"entity_hash":1,"entity_id":"homer"} ,"value":{"default:injury": [ { "key":{"time": "2016-01-01 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "arm" , "severity": 4 } }]} , { "key":{"time": "2016-01-02 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "arm" , "severity": 4 } }]} , { "key":{"time": "2016-01-03 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "arm" , "severity": 4 } }]} , { "key":{"time": "2016-01-04 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"some":"ignore"} , "location": "torso" , "severity": 3 } }]} , { "key":{"time": "2016-01-05 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "torso" , "severity": 5 } }]} ]}}
+{"key":{"entity_hash":2,"entity_id":"marge"} ,"value":{"default:injury": [ { "key":{"time": "2016-01-01 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "leg" , "severity": 4 } }]} , { "key":{"time": "2016-01-02 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "leg" , "severity": 4 } }]} , { "key":{"time": "2016-01-03 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "leg" , "severity": 4 } }]} , { "key":{"time": "2016-01-04 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"some":"ignore"} , "location": "head" , "severity": 3 } }]} , { "key":{"time": "2016-01-05 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "head" , "severity": 5 } }]} ]}}
+{"key":{"entity_hash":3,"entity_id":"moe"} ,"value":{"default:injury": [ { "key":{"time": "2016-01-01 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "ear" , "severity": 1 } }]} , { "key":{"time": "2016-01-02 00:00:00", "factset_id": 1} , "value": [ {"some": { "action": {"none":{}} , "location": "hair" , "severity": 2 } }]} ]}}
```

Comment:
yep, it's the zebra text format, I had to change the file from dates to times because the transmute isn't smart enough yet, was easier to just change the example at this point